src/chord/number.rs

```diff
@@ -9,17 +9,25 @@ const REGEX_NUMBER_NINTH: &str = "(?i)(ninth)";
 const REGEX_NUMBER_ELEVENTH: &str = "(?i)(eleventh)";
 const REGEX_NUMBER_THIRTEENTH: &str = "(?i)(thirteenth)";
 
+/// The superscript number after a chord.
 #[derive(Display, Debug, Clone, Copy, PartialEq)]
 pub enum Number {
+    /// A triad chord.
     Triad,
+    /// A seventh chord.
     Seventh,
+    /// A major seventh chord.
     MajorSeventh,
+    /// A ninth chord.
     Ninth,
+    /// An eleventh chord.
     Eleventh,
+    /// A thirteenth chord.
     Thirteenth,
```

Comment:
These comments seem to be unnecessary as they are the exact copies of the enum variables.